src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -551,6 +551,22 @@ static grpc_lb_addresses *process_serverlist_locked(
   return lb_addresses;
 }
 
+static grpc_lb_addresses *process_fallback_backend_addresses_locked(
```

Comment:
Since we now need our own custom function to create a copy of the address list, let's do the filtering for backend addresses at the same time.  In other words, let's do the following:

1. Change this function to only copy addresses with `is_balancer` set to false.
2. In the places where you are currently calling `grpc_lb_addresses_copy()` with the second parameter set to true, use this function instead.
3. In the places where you are currently using this function, use `grpc_lb_addresses_copy()`.
4. Revert the changes we made to lb_policy_factory.[ch] to add the second parameter to `grpc_lb_addresses_copy()`.

Reply:
That makes a lot of sense.